### PR TITLE
Create templatecache with webpack

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: contribs/gmf/src/gmf.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ goog.provide('gmf');
  * The default template base URL for modules, used as-is by the template cache.
  * @type {string}
  */
-gmf.baseModuleTemplateUrl = 'gmf';
+gmf.baseModuleTemplateUrl = 'gmf'; // nowebpack
```
Comment (ger-benjamin) [inline contribs/gmf/src/gmf.js]: You must remove also the comment

File: contribs/gmf/src/mobile/measure/lengthComponent.js [hunk 2]
```diff
@@ -18,16 +18,21 @@ gmf.mobile.measure.lengthComponent = angular.module('gmfMobileMeasureLength', [
...
+   * @param {angular.JQLite} element Element.
+   * @param {angular.Attributes} attrs Attributes.
+   * @return {string} The template url.
+   */
   (element, attrs) => {
     const templateUrl = attrs['gmfMobileMeasureLengthTemplateurl'];
     return templateUrl !== undefined ? templateUrl :
-      `${gmf.baseModuleTemplateUrl}/mobile/measure/lengthComponent.html`;
+      `${gmf.baseModuleTemplateUrl}/mobile/measure/lengthComponent.html`; // nowebpack
+    // webpack: 'gmf/measure/lengthComponent';
```
Comment (ger-benjamin) [inline contribs/gmf/src/mobile/measure/lengthComponent.js]: missing two spaces (indentation)

File: contribs/gmf/src/mobile/measure/pointComponent.js [hunk 3]
```diff
@@ -21,16 +21,21 @@ gmf.mobile.measure.pointComponent = angular.module('gmfMobileMeasurePoint', [
...
+   * @param {angular.JQLite} element Element.
+   * @param {angular.Attributes} attrs Attributes.
+   * @return {string} The template url.
+   */
   (element, attrs) => {
     const templateUrl = attrs['gmfMobileMeasurePointTemplateurl'];
     return templateUrl !== undefined ? templateUrl :
-      `${gmf.baseModuleTemplateUrl}/mobile/measure/pointComponent.html`;
+      `${gmf.baseModuleTemplateUrl}/mobile/measure/pointComponent.html`; // nowebpack
+    // webpack: 'gmf/measure/pointComponent';
```
Comment (ger-benjamin) [inline contribs/gmf/src/mobile/measure/pointComponent.js]: here too

File: contribs/gmf/src/objectediting/component.js [hunk 4]
```diff
@@ -63,7 +68,8 @@ gmf.objectediting.component.value('gmfObjecteditingTemplateUrl',
   ($element, $attrs) => {
     const templateUrl = $attrs['gmfObjecteditingTemplateurl'];
     return templateUrl !== undefined ? templateUrl :
-      `${gmf.baseModuleTemplateUrl}/objectediting/component.html`;
+      `${gmf.baseModuleTemplateUrl}/objectediting/component.html`; // nowebpack
+    // webpack: 'gmf/objectediting';
```
Comment (ger-benjamin) [inline contribs/gmf/src/objectediting/component.js]: here too

File: contribs/gmf/src/print/component.js [hunk 5]
```diff
@@ -34,6 +34,24 @@ gmf.print.component = angular.module('gmfPrintComponent', [
...
+  /**
+   * @param {angular.JQLite} element Element.
+   * @param {angular.Attributes} attrs Attributes.
+   * @return {string} Template.
+   */
+  (element, attrs) => {
+    const templateUrl = attrs['gmfPrintTemplateurl'];
+    return templateUrl !== undefined ? templateUrl :
+      `${gmf.baseModuleTemplateUrl}/print/component.html`; // nowebpack
+    // webpack: 'gmf/print';
```
Comment (ger-benjamin) [inline contribs/gmf/src/print/component.js]: here too

File: contribs/gmf/src/profile/component.js [hunk 6]
```diff
@@ -38,9 +38,14 @@ gmf.profile.component.value('gmfProfileTemplateUrl',
   ($element, $attrs) => {
     const templateUrl = $attrs['gmfProfileTemplateurl'];
     return templateUrl !== undefined ? templateUrl :
-      `${gmf.baseModuleTemplateUrl}/profile/component.html`;
+      `${gmf.baseModuleTemplateUrl}/profile/component.html`; // nowebpack
+    // webpack: 'gmf/profile';
```
Comment (ger-benjamin) [inline contribs/gmf/src/profile/component.js]: here too, please verify also the other

File: src/ngeo.js [hunk 7]
```diff
@@ -43,4 +43,4 @@ goog.require('ol.style.AtlasManager'); // nowebpack
  * The default template base URL for modules, used as-is by the template cache.
  * @type {string}
  */
-ngeo.baseModuleTemplateUrl = 'ngeo';
+ngeo.baseModuleTemplateUrl = 'ngeo'; // nowebpack
```
Comment (ger-benjamin) [inline src/ngeo.js]: I think you can delete the whole file and remove all the `// nowebpack` here

Comment (ger-benjamin) [reply src/ngeo.js]: And that mean, that you can also remove all the "goog.require('ngeo')" in component, on webpack

File: contribs/gmf/src/gmf.js (hunk 1 above)
Comment (sbrunner) [reply contribs/gmf/src/gmf.js]: I try without success ...

File: contribs/gmf/src/mobile/measure/lengthComponent.js (hunk 2 above)
Comment (sbrunner) [reply contribs/gmf/src/mobile/measure/lengthComponent.js]: => lint with get an error ...

**Pull request merged**